app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -1348,6 +1348,18 @@ class DatabaseMigrationAndroidTest {
     db_v30.assertColumnCount(tableName, expectedColumnCount)
```

Comment:
@vinaysshenoy sorry for not breaking this PR into two smaller chunks, but I wanted you to review these two commits together:

- `Remind user to add missing phone number`
- `Add MissingPhoneReminder table to database`

and they constitute the majority of LOC.

Reply:
They mostly include a duplicated dialog and a DB migration so I'm hoping reviewing it shouldn't be time consuming. Let me know if you still think I should break these two.